docs/modeling/physical_models.rst

```diff
@@ -77,15 +77,6 @@ give the wavelength and frequency of the maximum for :math:`B_\lambda`
 and :math:`B_\nu`, respectively, calculated using `Wein's Law
```

Comment:
```suggestion
and :math:`B_\nu`, respectively, calculated using `Wien's Law
```
Completely unrelated, just spotted (or rediscovered...) the typo here.

Reply:
Ops. Good catch!